renderdoc/driver/vulkan/wrappers/vk_device_funcs.cpp

```diff
@@ -2580,6 +2598,7 @@ bool WrappedVulkan::Serialise_vkCreateDevice(SerialiserType &ser, VkPhysicalDevi
       CheckDeviceExts();
     }
 
+    InitInstanceExtensionTables(m_Instance, &m_EnabledExtensions);
```

Comment:
Why is this being added here?

Reply:
I probably missed something somewhere else, but if I don't have that, one of the instance level vfunc is NULL and it crashes when trying to list the counters.

Reply:
Hmm, I'll need to check into this. I'd guess it's related to this extension being a _device_ extension but having an _instance_ level dispatch function, though there have been other extensions that do that e.g. `VK_EXT_sample_locations`. They don't break that I know of.

Reply:
OK yeh seems like while I handled this properly on capture, I didn't handle it on replay. The dispatch table for the instance is initialised before we know the enabled device extensions so any instance-dispatch functions tied to device extensions didn't get enabled.

The previous few cases where this came up we never needed to call the physical device functions on replay so this didn't crash. The fix is fine, I just wanted to make sure it was clear what this was fixing :).